src/apps/uclient/startuclient.c

```diff
@@ -473,11 +475,10 @@ static int clnet_allocate(int verbose, app_ur_conn_info *clnet_info, ioa_addr *r
             }
             {
               int found = 0;
+              for (stun_attr_ref sar = stun_attr_get_first(&response_message); sar;
```

Comment:
This is nice. clang-format did that?
Hope no one complains because of older compilers